Sources/MongoSwift/Operations/AggregateOperation.swift

```diff
@@ -72,7 +72,7 @@ public struct AggregateOptions: Codable {
         self.collation = collation
         self.comment = comment
         self.hint = hint
-        self.let = `let`
+        self.`let` = `let`
```

Comment:
ditto comment about whether we need backticks on left? (this seemed to compile/pass tests fine before?)